go/common/workerpool/workerpool.go

```diff
@@ -147,6 +213,16 @@ func New(name string) *Pool {
 		logger:       logging.GetLogger(fmt.Sprintf("workerpool/%s", name)),
 	}
 
+	if cfg != nil && cfg.Backoff != nil {
+		backoff := cmnBackoff.NewExponentialBackOff()
```

Comment:
I would move this to `newExpBackoff(cfg *BackoffConfig)` or `newExpBackoff(minTimeout time.Duration, maxTimeout time.Duration)` and as said before, maybe change the method to accept the backoff instead of creating one.